lib/ExtUtils/MakeMaker.pm

```diff
@@ -525,7 +525,10 @@ sub new {
                     # simulate "use warnings FATAL => 'all'" for vintage perls
                     die @_;
                 };
-                version->new( $perl_version )->numify;
+                my $v = version->new($perl_version);
+                # we care about parse issues, not numify warnings
+                no warnings;
```

Comment:
`no warnings "numeric"` may be better here.

Reply:
It would have to be `no warnings $] > 5.015 ? 'version' : 'numeric';`. I didn't think it would be worth it. There is really nothing `->numify` could ever warn about that we would ever care about, given that it already succeeded in parsing the version.

Reply:
I see, fair enough